suite-native/module-check-backup/src/hooks/useNavigateCheckBackup.ts

```diff
@@ -0,0 +1,40 @@
+import { useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { selectDeviceModel } from '@suite-common/wallet-core';
+import {
+    DeviceCheckBackupStackRoutes,
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { DeviceModelInternal, models } from '@trezor/device-utils';
+
+const checkBackupUnsupportedDeviceModels: Array<DeviceModelInternal> = [DeviceModelInternal.T1B1];
```

Comment:
nit: I think we're using this as a general code style thing, but again, just a nit that can be resolved
```suggestion
const checkBackupUnsupportedDeviceModels: DeviceModelInternal[] = [DeviceModelInternal.T1B1];
```

Reply:
fix: https://github.com/trezor/trezor-suite/pull/20014/commits/cf0cde4b882eb67a5e73412724e3cb121e696790  unintentionally committed it along with the feature flag implementation.

Btw: I keep changing this style with every project! 😄
Why don’t TypeScript engineers understand that sometimes humans don’t want to have a choice?! 😄
Array vs [], interface vs type…